### PR TITLE
Conformance results for v1.10/containership

### PR DESCRIPTION
**Pull request by normanjoyner**
vendor: Containership
name: Containership Kubernetes Engine (CKE)
version: v3.0.0
website_url: https://containership.io
documentation_url: https://docs.containership.io
product_logo_url: https://cdn.containership.io/logos/containership/cs-vertical.svg
type: distribution
description: 'Containership Kubernetes Engine (CKE) is a multi-cloud provisioning tool that provides a simple to use UI and API for Kubernetes cluster lifecycle management in the cloud or on-premises.'

### CODE REVIEW

File: v1.10/containership/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+vendor: Containership
+name: Containership Kubernetes Engine (CKE)
+version: v3.0.0
+website_url: https://containership.io
+documentation_url: https://docs.containership.io
+product_logo_url: https://cdn.containership.io/logos/containership/cke.svg
```
Comment (dankohn) [inline v1.10/containership/PRODUCT.yaml]: Could you please link to a version of your logo that says "Containership" underneath, as https://landscape.cncf.io/landscape=non-certified-kubernetes&selected=containership does. Or we can add it for you if you tell us the font to use.

Comment (normanjoyner) [reply v1.10/containership/PRODUCT.yaml]: updated @dankohn

Comment (dankohn) [reply v1.10/containership/PRODUCT.yaml]: Great.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**